cmd/podman/common.go

```diff
@@ -401,6 +401,14 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 		"publish-all", "P", false,
 		"Publish all exposed ports to random ports on the host interface",
 	)
+	createFlags.Bool(
+		"pull-always", false,
+		"Always pull the image from the source repository",
+	)
+	createFlags.Bool(
+		"pull", false,
```

Comment:
Shouldn't this be True?

Reply:
I believe so, at least that's it's setting in Buildah

Reply:
It would add a tad of overhead in some cases, so i'd default to false. If anyone feels strongly otherwise, doesn't matter to me.

Reply:
Well we are looking for the correct behaviour. 
--pull in `buildah from` and podman build` defaults to true.

```
podman build --help | grep pull
      --pull                                       pull the image if not present (default true)
```

Reply:
it should be false, having it check for a new image prior to running will eat time.

Reply:
What does docker do?  Does having it be pull=false, not pull if it is not local?  If not then this is different then podman build, if that is the case, we should use a different name --update?

Reply:
I honestly can't even figure out how to set up upstream docker to test with --pull ...